src/components/Problems/UploadProblemForm.tsx

```diff
@@ -61,6 +69,7 @@ export const UploadProblemForm: FC<{
               {acceptedFiles[0].name} ({niceBytes(acceptedFiles[0].size)})
             </span>
           )}
+          {fileRejections.length > 0 && <span className="error">Nahraný súbor musí byť vo formáte pdf.</span>}
```

Comment:
nevyzera, ze ten classname na nieco je
```suggestion
          {fileRejections.length > 0 && <span>Nahraný súbor musí byť vo formáte pdf.</span>}
```